File: app/src/main/java/com/example/storeapp/Activities/LoginActivity.java
package com.example.storeapp.Activities;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.storeapp.databinding.LoginLayoutBinding;

public class LoginActivity extends AppCompatActivity {

    private LoginLayoutBinding binding = null;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = LoginLayoutBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        // 1. Check inform user then to login
        binding.loginBtn.setOnClickListener(v -> {
            Intent intent = new Intent(this, MainActivity.class);
//            Bundle bundle = new Bundle();

            // 1.1 if login successful, go to main activity
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/example/myapplication/Activities/LoginActivity.kt
package com.example.myapplication.Activities

import android.content.Intent
import android.os.Bundle

import androidx.appcompat.app.AppCompatActivity

import com.example.myapplication.databinding.LoginLayoutBinding

class LoginActivity: AppCompatActivity() {

    private lateinit var binding: LoginLayoutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = LoginLayoutBinding.inflate(layoutInflater)
        setContentView(binding.root)

        // 1. Check inform user then to login
        binding.loginBtn.setOnClickListener(){
            val intent = Intent(this, MainActivity::class.java)
            val bundle = Bundle()

            startActivity(intent)
        }
    }
}